.github/workflows/image-scan-analysis.yml

```diff
@@ -1,61 +1,705 @@
-name: Security CI
+name: Image Security Scan
 
 on:
   push:
-    branches: [master, test, dev]
+    branches: [master, test]
   pull_request:
-    branches: [master, test, dev]
+    branches: [master, test]
   schedule:
-    - cron: '42 15 * * 5'
+    - cron: '30 5 14 * *' # Run this scan every sprint release cycle 14th of every month at 5:30 UTC
 
 jobs:
 
-  build_frontend:
+  nodejs-base-image:
+
+    name: nodejs-base-image
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+    env:
+      working-directory: ./openshift/4.0/templates/base-images
+      image-name: nodejs-14-ubi8
+      tag: ${{secrets.NODEJS14_BASE_TAG}} # 1-35 
+      OPENSHIFT_NAMESPACE: 3cd915-tools
+      output-filename: nodejs14.txt
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Authenticate and set context
+      uses: redhat-actions/oc-login@v1
+      env:
+      # These can be stored in secrets, if desired.
+        OPENSHIFT_USER: ${{secrets.OC_USERNAME_SA}} # service account gitaction in openshift will be used here!
+        OPENSHIFT_NAMESPACE: 3cd915-tools
+        SERVER_URL: ${{secrets.OPENSHIFT_SERVER}}
+        OPENSHIFT_REGISTRY: ${{secrets.OPENSHIFT_REGISTRY}} 
+
+      with:
+      # URL to your OpenShift cluster.
+      # Refer to Step 2.
+        openshift_server_url: ${{ secrets.OPENSHIFT_SERVER }}
 
+      # Authentication Token. Can use username and password instead.
+      # Refer to Step 3.
+        openshift_token: ${{ secrets.OPENSHIFT_TOKEN }}
+    
+    - name: Login to OpenShift Container Registry
+      uses: docker/login-action@v1
+      with:
+        registry: ${{secrets.OPENSHIFT_REGISTRY}}
+        username: ${{ secrets.OC_USERNAME_SA }}
+        password: ${{ secrets.OPENSHIFT_TOKEN }}
+    
+    - name: Pull nodejs Base Image
+      env:
+        OPENSHIFT_NAMESPACE: 3cd915-tools
+      run: |
+        #docker login -u ${{env.OPENSHIFT_USER}} -p $(oc whoami -t) ${{env.OPENSHIFT_REGISTRY}}   
```

Comment:
probably can remove these comments?

Reply:
I think this could be removed as oc login is used before. Or remove the oc login step and use docker login instead. Either should work